scripts/build.py

```diff
@@ -1099,6 +1099,9 @@ def generate_build_directory():
 
 
 def build():
+    """It creates a third party directory where all JS and CSS files
+    dependencies minifes and store.
```

Comment:
Simply write -> `Creates a third party directory where all the dependencies of JS and Html files are stored.`